src/style.css

```diff
@@ -1 +1,60 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 > * {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(1n + 2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(1n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(1n + 3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(1n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(1n + 4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(1n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(1n + 5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--5 > * {
```

Comment:
Again, the use of the `*` selector is not recommended as it can impact performance. It would be better to use a class selector or another more specific selector here.